core/chains/evm/client/rpc_client.go

```diff
@@ -229,7 +234,7 @@ func (r *rpcClient) Dial(callerCtx context.Context) error {
 // Not thread-safe, pure dial.
 // DialHTTP doesn't actually make any external HTTP calls
 // It can only return error if the URL is malformed.
-func (r *rpcClient) DialHTTP() error {
+func (r *RPCClient) DialHTTP() error {
 	promEVMPoolRPCNodeDials.WithLabelValues(r.chainID.String(), r.name).Inc()
 	lggr := r.rpcLog.With("httpuri", r.ws.uri.Redacted())
```

Comment:
Looks like this got unintentionally changed from `r.http.uri` to `r.ws.uri` in `DialHTTP()`